lib/modules/manager/flux/common.ts

```diff
@@ -1,6 +1,6 @@
 import { regEx } from '../../../util/regex';
 
-export const systemManifestRegex = '(^|/)flux-system/gotk-components\\.yaml$';
+export const systemManifestRegex = '(^|/)flux-system/(?:.+/)*gotk-components\\.yaml$';
```

Comment:
```suggestion
export const systemManifestRegex = '(^|\\/)flux-system\\/(?:.+\\/)?gotk-components\\.yaml$';
```
that should be simpler and the same outcome

Reply:
Thanks! Is the escaping needed here? I don't think the forward slash is special, and this makes it slightly less readable, but feel free to add the commit directly to the branch if you prefer!

Reply:
yes, as the slash is the start and end delimiter for a regex. the reason why it works without escape is that the js regex engine is not strict about that fact. so please be strict and properly escape.

Reply:
Ah, I see your reasoning. Yes, the slash is special only in the context of regex literals, but there is no delimiter character at all in this context of a string pattern being passed to a regex constructor. I don't see it as a matter of engine strictness, but I applied your suggestion anyway. Thanks again for the review, and please let me know if you see any other issue with the PR!